### PR TITLE
Add Summit Schedule.

### PR DESCRIPTION
**Pull request by jberkus**
Adds in the schedule of selected sessions for the Kubevirt Summit.

Attn @fabiand @rmohr

### CODE REVIEW

**Comment by marceloamaral** [comment]
Looks great!
/ok-to-test
/lgtm

**Comment by cwilkers** [comment]
One little formatting issue: Howard Zhang's talk title got broken into two cells. Also, one of the headers is not capitalized like the others. (Register)

File: pages/summit.md [hunk 1]
```diff
@@ -11,31 +11,52 @@ The KubeVirt Summit is a 2-day virtual event to discover,
...
+
 
-You can [RSVP for KubeVirt Summit on community.cncf.io](https://community.cncf.io/events/details/cncf-kubevirt-community-presents-kubevirt-summit-2022-day-1/).  You will need to register with CNCF.io if you have not before. Attendance is free.
+## [DAY 1](https://community.cncf.io/events/details/cncf-kubevirt-community-presents-kubevirt-summit-2022-day-1/) Wednesday, February 16
 
-## Call for Session Proposals
+| Time (UTC) | Time (EST) | Session | Speaker | Track |
+| ------- | ------- | ----------------------------- | ------- | ------ | 
+| 14:00 | 9:00 | Keynote/Intro | Fabian Deutch | General |
+| 14:25 | 9:25 | "KubeVirt on Arm64 (Progress |  Troubles and Plans)" | Howard Zhang | Contributors |
```
Comment (cwilkers) [inline pages/summit.md:38]: The pipe between Progress and Troubles is turning into a column break, can it be a colon instead?

File: pages/summit.md [hunk 2]
```diff
@@ -11,31 +11,52 @@ The KubeVirt Summit is a 2-day virtual event to discover,
...
-
 ## When
 
 The event will take place online during two half-days (5 hours each day):
 
 - Dates: February 16 and 17, 2022
 - Time: 14:00 – 19:00 UTC (09:00–14:00 EST, 15:00–20:00 CET)
 
-## RSVP
+## register
```
Comment (cwilkers) [inline pages/summit.md:21]:
```suggestion
## Register
```

**Review by cwilkers — CHANGES_REQUESTED** [review]
I made more specific comments and suggestions in review.

File: pages/summit.md [hunk 3]
```diff
@@ -11,31 +11,52 @@ The KubeVirt Summit is a 2-day virtual event to discover,
...
+| 15:00 | 10:00 | Live Migration Policies - fine tuning migrations dynamically | Itamar Holder | Users |
+| 15:30 | 10:30 | Live Migration Policies - fine tuning migrations dynamically | Itamar Holder | Users |
+| 16:00 | 11:00 | KubeVirt+Harverster: a Windows user tale | Nuno do Carmo | Users |
+| 16:35 | 11:35 | Kubernetes clusters on KubeVirt VMs | David Vossel | Users |
+| 17:10 | 12:10 | Network interface hotplug for KubeVirt | Miguel Duarte Barroso | Contributors |
+| 17:45 | 12:45 | A few bugs and findings from VMI Churn at NVIDIA | Fan Zhang | Contributors |
+| 18:20 | 13:20 | EdgeStack and Singulus: Complete VM network implementation with KubeVirt CNI  | Sangho Shin, Jian Li | Users |
 
-We are [looking for community members to lead sessions](https://forms.gle/KXaPWpJzk7Z12bRD8) on contributing to, or using, KubeVirt.  Topics should be anything that is open source and focused on the KubeVirt ecosystem, whether contributing to it or using it. Examples of talks from last year's Summit include:
+## [DAY 2](https://community.cncf.io/events/details/cncf-kubevirt-community-presents-kubevirt-summit-2022-day-1/) Thursday, February 17
```
Comment (rmohr) [inline pages/summit.md:47]: Here should be `day-2` at the end of the link.

**Review by rmohr — CHANGES_REQUESTED** [review]
One  link is wrong, the content looks good to me. :+1:

**Pull request merged**